### PR TITLE
Improve withdrawals flow

### PR DESCRIPTION
**Pull request by marc-aurele-besner**
## Improve withdrawals flow

Split pending and ready withdrawals and add the action to unlock

### Screenshots

Withdrawls

### CODE REVIEW

File: explorer/src/components/Staking/MyWithdrawals.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+                  },
+                  {
+                    accessorKey: 'total.sortValue',
+                    header: 'Total Withdrawal Amount',
+                    enableSorting: true,
+                    cell: ({ row }) => <div>{row.original.total.value}</div>,
+                  },
+                  {
+                    accessorKey: 'unlockAtConfirmedDomainBlockNumber.sortValue',
+                    header: 'Withdrawal at Domain Block Number',
```
Comment (jfrank-summit) [inline explorer/src/components/Staking/MyWithdrawals.tsx:248]: Might be nice to have a current block number available, or maybe `Domain blocks until withdrawal`.

File: explorer/src/components/Staking/MyWithdrawals.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        }),
+    [withdrawals, subspaceAccount, selectedChain.token.symbol],
+  )
+
+  const myUnlockedWithdrawalsList = useMemo(
+    () =>
+      myUnlockedWithdrawals.length > 0 && (
+        <div className='mt-5 flex flex-col gap-2'>
+          <div className='text-base font-medium text-grayDark dark:text-white'>
+            My Unlocked Withdrawals ({myUnlockedWithdrawals.length})
```
Comment (jfrank-summit) [inline explorer/src/components/Staking/MyWithdrawals.tsx:73]: Are these actually unlocked already, or waiting to be unlocked? The label indicates they are already unlocked but there is still an action dropdown with `Unlock...` options so I assume they are waiting to be unlocked. 
image

Comment (jfrank-summit) [reply explorer/src/components/Staking/MyWithdrawals.tsx:73]: Also, as you can see from the image above, you can't read the whole action.

Comment (marc-aurele-besner) [reply explorer/src/components/Staking/MyWithdrawals.tsx:73]: Label is likely misleading, it's more Ready to be unlocked 👍

File: explorer/src/components/Staking/MyWithdrawals.tsx (hunk 1 above)
Comment (marc-aurele-besner) [reply explorer/src/components/Staking/MyWithdrawals.tsx:248]: What matter is the domain block number and we don't currently query multiple network data at ounce...
I just realized I can extract the last domain block number from the `BundleStored` event, so I'll add this to the staking-sqid domain entry

**Review by jfrank-summit — APPROVED** [review]

**Pull request merged**